cms/migrations/0004_facultymemberspage.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 2.2.10 on 2020-05-06 12:14
+
+from django.db import migrations, models
+import django.db.models.deletion
+import wagtail.core.blocks
+import wagtail.core.fields
+import wagtail.images.blocks
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wagtailcore', '0045_assign_unlock_grouppagepermission'),
+        ('cms', '0003_bootcamp_product_run_page'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='FacultyMembersPage',
+            fields=[
+                ('page_ptr', models.OneToOneField(auto_created=True, on_delete=django.db.models.deletion.CASCADE, parent_link=True, primary_key=True, serialize=False, to='wagtailcore.Page')),
+                ('sections', wagtail.core.fields.StreamField([('section', wagtail.core.blocks.StructBlock([('heading', wagtail.core.blocks.CharBlock(help_text='The heading to display for this section on the product page.', max_length=255)), ('subhead', wagtail.core.blocks.CharBlock(help_text='The subhead to display for this section on the product page.', max_length=255)), ('members', wagtail.core.blocks.StreamBlock([('member', wagtail.core.blocks.StructBlock([('name', wagtail.core.blocks.CharBlock(help_text='Name of the faculty member.', max_length=100)), ('image', wagtail.images.blocks.ImageChooserBlock(help_text='Profile image size must be at least 300x300 pixels.')), ('description', wagtail.core.blocks.RichTextBlock(help_text='A brief description about the faculty member.'))]))], help_text='The faculty members to display on this page'))]))], help_text='The faculty members to display on this page')),
```

Comment:
```suggestion
                ('sections', wagtail.core.fields.StreamField([('section', wagtail.core.blocks.StructBlock([('heading', wagtail.core.blocks.CharBlock(help_text='The heading to display for this section on the product page.', max_length=255)), ('subhead', wagtail.core.blocks.CharBlock(help_text='The subhead to display for this section on the product page.', max_length=255)), ('members', wagtail.core.blocks.StreamBlock([('member', wagtail.core.blocks.StructBlock([('name', wagtail.core.blocks.CharBlock(help_text='Name of the faculty member.', max_length=100)), ('image', wagtail.images.blocks.ImageChooserBlock(help_text='Profile image size must be at least 300x300 pixels.')), ('description', wagtail.core.blocks.RichTextBlock(help_text='A brief description about the faculty member.'))]))], help_text='The instructors to display in this section'))]))], help_text='The instructors to display in this section')),
```

Reply:
Also, this is a really long line. Did it pass the linter?

Reply:
Actually, its an auto-generated file by Django management commands. We don't change it.
It did not raise any warning or error. Will confirm it after rebasing as it already needs to be rebased to deal with the `nose` issue.